test/expected.txt

```diff
@@ -46,26 +48,30 @@
 || 
 || 0 persons listed!
 || ===================================================
+|| Enter command: || [Command entered:  sort]
+||
+|| 0 persons listed!
+|| ===================================================
 || Enter command: || [Command entered:  add wrong args wrong args]
-|| Invalid command format! 
+|| Invalid command format!
```

Comment:
Whitespace changes like this cause your tests to fail and also add unrelated changes to the PR. Always best to double check before committing.